com.unity.render-pipelines.universal/ShaderLibrary/UnityGBuffer.hlsl

```diff
@@ -159,9 +159,9 @@ BRDFData BRDFDataFromGbuffer(half4 gbuffer0, half4 gbuffer1, half4 gbuffer2)
     half smoothness = gbuffer2.a * 0.5h + 0.5h;
```

Comment:
Why is the smoothness packed (I assume [0..1] range as the target format is 8bit srgb, but of course alpha is linear). Why not use the whole range?